js/main.js

```diff
@@ -95,6 +121,7 @@ function handleNew(event) {
   if (data.view === 'entry-form') {
     switchView();
   }
+  data.editing = null;
```

Comment:
You should move this to your `submit` event handler, it can be the last action of the function